sashimi-webapp/src/logic/filemanager/data/file.js

```diff
@@ -1,13 +1,12 @@
 import storage from 'src/database/storage';
 import Folder from './folder';
+import idMap from './idmap';
```

Comment:
Just some suggestion to improve code aesthetic. 
instead of a relative import `import idMap from './idmap'`,
can consider a consistent absolute import `import idMap from 'src/logic/filemanager/data/idmap'`
but not important for now.